Project.toml

```diff
@@ -12,11 +12,11 @@ Strided = "5e0ebb24-38b0-5f93-81fe-25c709ecae67"
 ZipFile = "a5390f91-8eb1-5f08-bee0-b1d1ffed6cea"
 
 [compat]
-DataStructures = "0.17, 0.18"
+DataStructures = "0.18"
 InternedStrings = "0.7"
-Strided = "1.0"
+Strided = "1.1"
 ZipFile = "0.9"
-julia = "1.6"
+julia = "1"
```

Comment:
Since Pickle is not compatible with 1.5, what I want here is to support nightly. The Julia version should be 1.6 and 1.7? Or I should keep it just "1"?